libs/template/config.go

```diff
@@ -36,6 +39,22 @@ func newConfig(ctx context.Context, schemaPath string) (*config, error) {
 	}, nil
 }
 
+func validateSchema(schema *jsonschema.Schema) error {
+	for _, v := range schema.Properties {
+		switch v.Type {
+		case jsonschema.NumberType, jsonschema.BooleanType, jsonschema.StringType, jsonschema.IntegerType:
+			continue
+		case jsonschema.ArrayType, jsonschema.ObjectType:
+			return fmt.Errorf("property type %s is not supported by bundle templates", v.Type)
+		case "int", "int32", "int64":
+			return fmt.Errorf("type %s is not a recognized json schema type. Please use \"integer\" instead", v.Type)
+		default:
+			return fmt.Errorf("type %s is not a recognized json schema type", v.Type)
+		}
+	}
+	return nil
```

Comment:
Please move this validation logic into the jsonschema package. Can be part of a `jsonschema.Load` function to read a file from disk, unmarshal it into the structure, and validate its contents.